borgmatic/config/schema.yaml

```diff
@@ -88,6 +88,13 @@ map:
                 type: scalar
                 desc: Umask to be used for borg create.
                 example: 0077
+            archive_name:
+                type: scalar
+                desc: |
+                    Name of the archive. Borg placeholders can be used, see
+                    https://borgbackup.readthedocs.io/en/stable/usage.html#borg-help-placeholders
+                    Default is "{hostname}-{now:%Y-%m-%dT%H:%M:%S.%f}"
+                example: "Documents-{hostname}-{now}"
```

Comment:
When I think of a label like "Documents", I tend to think of it as logically coming *after* hostname.. My rationale is that the archive name format is sort of a series of namespaces from least specific to most specific. E.g., first hostname, then the label for the thing being backed up on that particular host, and finally the timestamp of when it was backed up. So I would suggest an example like "{hostname}-documents-{now}" or similar.

One implication of that change is that when matching on a prefix (for pruning), the prefix would have to contain either "{hostname}" or "{hostname}-documents". That might be totally okay.

However, I do not feel super strongly about this point.